### DISCUSSION TITLE
Multiple releases of the same package each time there is a new version of PHP releases

### DISCUSSION THREAD

**Discussion by Alwaysin** [oerdnj/deb.sury.org#2120] [General]
Hello,

It seems that each time a new version is released, the same package is built three times, at several days apart. Excerpt from apt log on a specific package to keep it readable:

```
Start-Date: 2024-04-14  09:16:02
Upgrade: php8.0-soap:amd64 (1:8.0.30-2+0~20230904.59+debian11~1.gbp806e95, 1:8.0.30-5+0~20240411.61+debian11~1.gbpb0e95a)
Start-Date: 2024-04-19  07:12:50
Upgrade: php8.0-soap:amd64 (1:8.0.30-5+0~20240411.61+debian11~1.gbpb0e95a, 1:8.0.30-5+0~20240416.62+debian11~1.gbpb0e95a)
Start-Date: 2024-04-22  07:36:09
Upgrade: php8.0-soap:amd64 (1:8.0.30-5+0~20240416.62+debian11~1.gbpb0e95a, 1:8.0.30-5+0~20240421.64+debian11~1.gbpb0e95a)
```

Is that a normal behaviour? Commit seems to always be the same, in this case gbpb0e95a, which makes me think it is the exact same package each time.

**Comment by oerdnj** [comment]
This was already explained here: https://github.com/oerdnj/deb.sury.org/issues/2101
